src/External/GLFW/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ set(GLFW_INSTALL OFF)
 option(GLFW_VULKAN_STATIC "Use the Vulkan loader statically linked into application" OFF)
```

Comment:
The current install behavior of GLFW is different than other libraries when ``BUILD_GLFW`` switch is ``ON``. I have done some initial testing, I think the solution here is to change:
```
set(GLFW_INSTALL OFF)
```
to
```
set(GLFW_INSTALL ON)
```
Then its install behavior is consistent with other libraries (i.e., glfw3.lib is copied to the install directory of Open3D).

Reply:
I have also tried this option and it didn't work for some reason.
I will double check.

Reply:
I found out why this wasn't working for me: I was setting GLFW_INSTALL in src/CMakeLists.txt and the value was overridden in src/External/GLFW/CMakeLists.txt.